libs/coin-framework/src/bot/types.ts

```diff
@@ -221,3 +222,37 @@ export type MinimalSerializedReport = {
   environment: string | undefined;
   seedHash: string;
 };
+
+export type State<T extends TransactionCommon> = {
```

Comment:
Could be argued that these are internal types, as opposed to types that are exposed to users of the bot.
But no big deal, let's have them here.